Prevent stripping of VST3 on Gentoo

This patch felt like the most proper way to remove the QA warnings, haha.

Patch by Ryan Fox

--- a/scripts/linux/package-vst3.sh
+++ b/scripts/linux/package-vst3.sh
@@ -17,4 +17,3 @@ fi
 VST_SO_DIR="$BUNDLE_DIR/Contents/${ARCH}-linux"
 mkdir -p "$VST_SO_DIR"
 cp $1 "$VST_SO_DIR"/Surge.so
-strip -s "$VST_SO_DIR"/Surge.so
